docker-compose.override.prod.yml

```diff
@@ -37,11 +37,3 @@ services:
       - setup.kibana.host=kibana:5601
       - output.elasticsearch.hosts=elasticsearch:9200
       - output.elasticsearch.enabled=true
-
```

Comment:
No Kibana in production ?

Reply:
my bad, I didn't realize I removed it from prod. we just need it in dev too :)